cupy/core/core.pyx

```diff
@@ -2743,14 +2745,23 @@ not_equal = create_comparison(
 cpdef ndarray _convert_object_with_cuda_array_interface(a):
     cdef Py_ssize_t sh, st
     desc = a.__cuda_array_interface__
+    # TODO(leofang): enforce compliance to the latest protocol?
     shape = desc['shape']
     dtype = numpy.dtype(desc['typestr'])
+    if 'mask' in desc:
+        mask = desc['mask']
+        if mask is not None:
+            raise ValueError("CuPy currently does not support masked arrays.")
```

Comment:
Please use single quotes for consistency.

Reply:
```suggestion
            raise ValueError('CuPy currently does not support masked arrays.')
```